services_app/src/androidTest/java/org/opendatakit/database/service/ODKServiceTestRule.java

```diff
@@ -234,8 +234,9 @@ void shutdownService() throws TimeoutException {
          } catch (InterruptedException e) {
             e.printStackTrace();
          }
-         if(!conn.binderIsNull()) {
-            InstrumentationRegistry.getContext().unbindService(conn);
+          assert conn != null;
+          if(!conn.binderIsNull()) {
+             ApplicationProvider.getApplicationContext().unbindService(conn);
```

Comment:
I believe you are getting the wrong context here. The test used to get the "instrumentation/testing" context not the application under test.